feat(blog): add post on ITC25 presentations (#263)
freelawproject/free.law: posts/2025/02/26/free-law-project-at-itc.mdx

---
title: "Free Law Project at ITC: Tech, Access to Justice, and Big Ideas"
date: "2025-02-26"
tags:
 - "ITC25"
 - "E-filing"
 - "Michael Lissner"
 - "Jenifer Whiston"
 - "Jessica Frank"
 - "Free Law Project"
 - "LSC"
 - "Legal Services Corporation"
 - "open source"
author: "Jessica Frank and Jenifer Whiston"

> excerpt: "The Free Law Project team recently attended the Legal Services Corporation’s Innovations in Technology Conference, where we explored cutting-edge ways to improve access to justice through technology. By presenting on topics like e-filing solutions and free legal research tools, we showcased innovative collaborations and underscored the transformative potential of inclusive, user-centered tech in addressing the justice gap."

Reviewer: I agree this is long. Perhaps:

```suggestion
excerpt: "At LSC technology conference we presented on e-filing solutions, free legal research tools, innovative collaborations, and the transformative potential of inclusive, user-centered tech in addressing the justice gap."
```

---

> <p className="lead">In January, the Free Law Project team had an exciting time at the Legal Services Corporation’s Innovations in Technology Conference (ITC). This annual event brings together legal professionals, technologists, and advocates from across the country to explore how technology can improve access to justice. The energy was inspiring, and our team left feeling motivated about the future of legal tech.</p>

Reviewer: Usually, we do a pretty short lead, so:

```suggestion
<p className="lead">In January, the Free Law Project team had an exciting time at the Legal Services Corporation’s Innovations in Technology Conference (ITC).</p>

This annual event brings together legal professionals, technologists, and advocates from across the country to explore how technology can improve access to justice. The energy was inspiring, and our team left feeling motivated about the future of legal tech.
```
